pages/[id].tsx

```diff
@@ -88,14 +88,29 @@ const CandyMachine: NextPage = () => {
                     <span className='break-all border border-slate-300 shadow-xl py-2 px-4 rounded-lg text-center'>
                         {candyMachineAccount}{' '}
                     </span>
-                    <a
-                        className='text-[hsl(258,52%,56%)] mt-4'
-                        href={`https://solscan.io/account/${candyMachineAccount}?cluster=devnet`}
-                        target='_blank'
-                        rel='noopener noreferrer'
-                    >
-                        View in Solscan
-                    </a>
+                    <div className='mt-5'>
+                        <a
+                            className='text-[hsl(258,52%,56%)] mt-4 mr-2'
+                            href={`https://solscan.io/account/${candyMachineAccount}${
+                                connection.rpcEndpoint.includes('devnet') ? '?cluster=devnet' : ''
+                            }/`}
+                            target='_blank'
+                            rel='noopener noreferrer'
+                        >
+                            View in Solscan
+                        </a>
+                        -
+                        <a
+                            className='text-[hsl(258,52%,56%)] mt-4 ml-2'
+                            href={`https://solana.fm/address/${candyMachineAccount}${
+                                connection.rpcEndpoint.includes('devnet') ? '?cluster=devnet-solana' : ''
```

Comment:
Since we use this in many places, we can extract to a util function `useExplorer` (or another name)
that receives the account/transaction/token that we need to show, and the connection from useConnection,
function returns the full url with or without the `?cluster=devnet`